build/postgres-operator/Dockerfile

```diff
@@ -1,31 +1,8 @@
-ARG BASE_IMAGE_OS
-ARG DOCKERBASEREGISTRY
-FROM ${DOCKERBASEREGISTRY}${BASE_IMAGE_OS}
-
-ARG PGVERSION
-ARG PG_FULL
-ARG PACKAGER
-ARG RELVER
-
-MAINTAINER info@crunchydata.com
-
-LABEL vendor="Crunchy Data" \
-	url="https://crunchydata.com" \
-	release="${RELVER}" \
-	postgresql.version.major="${PGVERSION}" \
-	postgresql.version="${PG_FULL}" \
-	org.opencontainers.image.vendor="Crunchy Data" \
-	io.openshift.tags="postgresql,postgres,sql,nosql,crunchy" \
-	io.k8s.description="Trusted open source PostgreSQL-as-a-Service" \
-	name="postgres-operator" \
-	summary="Crunchy PostgreSQL Operator" \
-	description="Crunchy PostgreSQL Operator"
+FROM registry.access.redhat.com/ubi8/ubi-micro
 
 COPY licenses /licenses
 
-RUN ${PACKAGER} -y update && ${PACKAGER} -y clean all
```

Comment:
🤔 : Do you know what effect this line would have originally?

Reply:
It updates the packages installed by default on the base image, then cleans extra files, cache, packages, etc. Because we are using the micro image as the base, there are fewer default packages, meaning we don't need to update them. In general, the idea is to have the most updated packages when we build to reduce the number of CVEs we hit.